src/components/cards/configuration/PrintHistoryCard.vue

```diff
@@ -29,7 +29,7 @@ import { SocketActions } from '@/socketActions'
 })
 export default class PrinterHistoryCard extends Vue {
   handleRemoveAll () {
-    this.$confirm(this.$t('Are you sure? This will clear all history, and printer statistics.').toString())
+    this.$confirm(this.$t('app.history.msg.confirm').toString())
```

Comment:
Just FIY, this force the evaluation of the string, which also means it's not going to be reactive.